src/components/ChannelPreview/__tests__/ChannelPreview.test.js

```diff
@@ -0,0 +1,168 @@
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies
+import axios from 'axios';
+// eslint-disable-next-line import/no-extraneous-dependencies
```

Comment:
This [shouldn't be needed]( https://github.com/GetStream/stream-chat-react/blob/master/package.json#L68).